materialize-bigquery/.snapshots/TestSpecification

```diff
@@ -2,35 +2,50 @@
   "endpoint_spec_schema_json": {
     "$schema": "http://json-schema.org/draft-04/schema#",
     "required": [
+      "billing_project_id",
```

Comment:
I don't think this should be required. AFAIK it's completely optional, and most users won't need to specify it.